modules/spring-config/src/main/resources/META-INF/mule-core-common.xsd

```diff
@@ -1493,6 +1493,13 @@
                 <xsd:sequence>
                     <xsd:element ref="scheduling-strategy" minOccurs="1" maxOccurs="1"/>
                 </xsd:sequence>
+                <xsd:attribute name="synchronous" default="false" type="xsd:string">
```

Comment:
Can we use something clearer? `synchronous` makes it seem that it has to do with threading.  Are we sure best default behaviour is false?

Reply:
`false` as behavior is the current behaviour in 4.0.0, so we don't break compatibility. Let me think about the name.